stage0/Cargo.toml

```diff
@@ -20,3 +20,8 @@ static_assertions = "*"
 strum = { version = "*", default-features = false, features = ["derive"] }
 x86_64 = "*"
 zerocopy = "*"
+p384 = { version = "0.13.0", default-features = false, features = ["ecdsa"]}
+hkdf = {version = "0.12.3", default-features = false}
+rand_core = {version = "0.6.4", default-features = false, features = ["getrandom"]}
+hex = { version = "0.4", default-features = false, features = ["alloc"] }
+coset = {version = "0.3.4", default-features = false}
```

Comment:
nit: missing blank line at the end of some of the files. You can fix formatting issues by running `./scripts/xtask format`.

Reply:
done.